EPlast/EPlast.WebApi/Controllers/ClubController.cs

```diff
@@ -107,7 +107,12 @@ public async Task<IActionResult> GetNotActiveClubs(int page, int pageSize, strin
             return Ok(citiesViewModel);
         }
 
-        
+        [HttpGet("GetUserAccesses/{userId}")]
```

Comment:
First of all, why it is named as `GetUserAccesses` if it only get 1 access for one user?(it is what I get from method name and method itself)
And also don't think that this HTTP route is appropriate
You can`t just copy it from method name
Please, consider this route: 
```suggestion
        [HttpGet("user/access/{userId}")]
```